app/models/notifications/queries/UcrInfo.scala

```diff
@@ -24,5 +26,10 @@ abstract class UcrInfo {
   val entryStatus: Option[EntryStatus]
   val movements: Seq[MovementInfo]
 
-  def transport: Option[Transport] = movements.find(_.transportDetails.isDefined).flatMap(_.transportDetails)
+  def transport: Option[Transport] =
```

Comment:
can this be a lazy val also - no need to run this expensive computation more than once.